lib/kg_web/templates/page/index.html.eex

```diff
@@ -140,3 +140,5 @@
   </section>
 
 </div>
+
+<script src="<%= Routes.static_path(@conn, "/js/animations.js") %>"></script>
```

Comment:
moved the script tag here from app.html because I only want it run on the home page, not all pages.